client/src/render.rs

```diff
@@ -98,7 +98,7 @@ impl RenderPlugin {
         let camera_transf = engine.ecs().get::<Transform>(camera_entity).unwrap();
         let camera_comp = engine.ecs().get::<CameraComponent>(camera_entity).unwrap();
         let proj = camera_comp.projection[camera_idx];
-        let view = camera_transf.view() * vr_view;
+        let view = vr_view * camera_transf.view();
```

Comment:
Is there a difference between these changes or is this just aesthetic?

Reply:
There is! Matrix multiplication is (in almost all cases) non-commutative. This fixed an issue where the VR view was not coupled to the motion of the headset if the camera position wasn't at the origin